charmhub/info.go

```diff
@@ -56,7 +57,7 @@ func (c *InfoClient) Info(ctx context.Context, name string) (transport.InfoRespo
 		}
 		return resp, resultErr
 	}
-
+	c.logger.Tracef("Info() unmarshalled: %s", pretty.Sprint(resp))
```

Comment:
Should we wrap in `if c.logger.IsTraceEnabled() { ... }` to avoid pretty-printed a large object to a string in the normal case (when we're not tracing)?

Reply:
Had a discussion with some folks.  In general is not needed in the case.  Though there can be other cases were it is.